packages/@react-aria/i18n/src/useDefaultLocale.ts

```diff
@@ -28,6 +28,12 @@ export interface Locale {
 export function getDefaultLocale(): Locale {
   // @ts-ignore
   let locale = (typeof navigator !== 'undefined' && (navigator.language || navigator.userLanguage)) || 'en-US';
+  try {
+    // @ts-ignore
+    Intl.DateTimeFormat.supportedLocalesOf([locale]);
```

Comment:
Maybe we could normalize using `Intl.getCanonicalLocales()`? Seems to have pretty wide browser support, and made for this purpose: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/getCanonicalLocales

Reply:
it has worse browser support than [DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/supportedLocalesOf) though, I went with this one especially because of it

Reply:
hmm, well it does cover all of React Aria's [supported browsers](https://react-spectrum.adobe.com/react-aria/interactions.html#supported-browsers-and-platforms), but this is ok for now.